notify/mail.sh

```diff
@@ -10,6 +13,69 @@ mail_send() {
   _debug "_content" "$_content"
   _debug "_statusCode" "$_statusCode"
 
-  _err "Not implemented yet."
-  return 1
+  if _exists "sendmail"; then
+    _MAIL_BIN="sendmail"
+  elif _exists "ssmtp"; then
+    _MAIL_BIN="ssmtp"
+  elif _exists "mutt"; then
+    _MAIL_BIN="mutt"
+  elif _exists "mail"; then
+    _MAIL_BIN="mail"
+  else
+    _err "Please install sendmail, ssmtp, mutt or mail first."
+    return 1
+  fi
+
+  MAIL_FROM="${MAIL_FROM:-$(_readaccountconf_mutable MAIL_FROM)}"
+  if [ -z "$MAIL_FROM" ]; then
+    MAIL_FROM="$USER@$(hostname -f)"
+    _info "The MAIL_FROM is not set, so use the default value: $MAIL_FROM"
+  fi
+  _saveaccountconf_mutable MAIL_FROM "$MAIL_FROM"
+
+  MAIL_TO="${MAIL_TO:-$(_readaccountconf_mutable MAIL_TO)}"
+  if [ -z "$MAIL_TO" ]; then
+    MAIL_TO="$(_readaccountconf ACCOUNT_EMAIL)"
```

Comment:
`ACCOUNT_EMAIL`  can be empty too.
please check and throw error here.

Reply:
- Both e-mails are now validated,
- MAIL_FROM is not required,
- MAIL_TO is not saved if ACCOUNT_EMAIL is used.